lookout/style/format/langs/__main__.py

```diff
@@ -27,7 +35,13 @@ def parse_args():
     # 2. "glob"-style filtering
     #
     # Use find | xargs instead.
-    parser.add_argument("input", nargs="+", help="Paths to the sample files.")
+    parser.add_argument("-i", "--input", nargs="+", help="Paths to the sample files.")
+    parser.add_argument("--parquet", action="store_true", default=False,
+                        help="Use parquet files as input. "
+                             "Should contain next columns: 'path', 'content', 'uast'")
+    languages = ["java", "python", "go", "javascript", "typescript", "ruby", "bash", "php"]
+    parser.add_argument("-l", "--language", choices=languages, default="",
```

Comment:
Rename to `--parquet-language` and say explicitly that the option is ignored if there is no `--parquet`

Reply:
but it is not. you can specify it. Before we have random language in files collection in case there are several of them. and now you can specify it directly. I think it is more convenient.